.github/workflows/django.yml

```diff
@@ -82,6 +82,14 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.COV }}
         run: |
           coveralls
+              - name: Run Frontend Tests
+        run: |
```

Comment:
just saw that building with github actions failed. seems to be line 85 is indented too much.